test/src/helpers/util.test.js

```diff
@@ -11,3 +13,38 @@ describe('util', () => {
     });
   });
 });
+
+describe('crypto-hash', () => {
```

Comment:
You should put the tests in the same folder structure as the source code 'test/src/helpers/hash/crypto.test.js'